src/components/container/ConnectFitbit/ConnectFitbit.stories.tsx

```diff
@@ -40,14 +40,6 @@ NotEnabledHide.args = { previewState: "notEnabled", title: "Fitbit", disabledBeh
 export const NotEnabledDisplayError = Template.bind({});
 NotEnabledDisplayError.args = { previewState: "notEnabled", title: "Fitbit", disabledBehavior: "displayError" };
 
-export const HideConnectedFetchComplete = Template.bind({});
-HideConnectedFetchComplete.args = { previewState: "fetchComplete", title: "Fitbit", hideWhenConnected: true };
-
-export const HideConnectedNotConnected = Template.bind({});
-HideConnectedNotConnected.args = { previewState: "notConnected", title: "Fitbit", hideWhenConnected: true };
-
-export const HideConnectedUnauthorized = Template.bind({});
-HideConnectedUnauthorized.args = { previewState: "unauthorized", title: "Fitbit", hideWhenConnected: true };
-
-export const HideConnectedFetchingData = Template.bind({});
-HideConnectedFetchingData.args = { previewState: "fetchingData", title: "Fitbit", hideWhenConnected: true };
+export const HideConnected = Template.bind({});
+HideConnected.args = { previewState: "fetchComplete", hideWhenConnected: true };
+HideConnected.argTypes = { previewState: { name: "Connection State", control: "radio", options: ["notConnected", "notEnabled", "unauthorized", "error", "fetchComplete", "fetchingData"]}};
```

Comment:
Are notEnabled and error new?

Reply:
The control currently has display messages for the `fetchComplete`, `fetchingData`, and `unauthorized`.  There is no specific message displayed for an `error` state.

I'm not sure if this is a state that is intended to be reached - or what the should be displayed if it is - as there is currently no logic to display anything particular for an "error" state.  I've removed the logic handling the "error" state, as there are no stories regarding it in any configuration.  

@greinard Did you have any feedback on that the `error` state?

Reply:
I think we should probably treat the `error` state as one that "needs attention" and therefore causes the widget to show up, including when `hideWhenConnected` is `true`.  If we are not displaying an error message of some kind on that widget when it is in the `error` state, perhaps we should be?

It is a valid status from the server: https://github.com/CareEvolution/Consumers/blob/master/Mogul/Libraries/RKStudioLibrary/Models/ExternalAccountPrimitives.cs#L36-L42

And we treat it as "needing attention" in the Connect EHR widget: https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/container/ConnectEhr/ConnectEhr.tsx#L63